docs/Developers/Newcomers_Guide.rst

```diff
@@ -85,8 +92,8 @@ Once you complete this, please take the time and
 Step 1. Meet the Community!
 ---------------------------
 
-To get started, the first step is to meet the community. We use gitter to
-communicate, and there the helpful community will guide you.
+To get started, the first step is to meet the community.
+We use gitter to communicate, and there the helpful community will guide you.
```

Comment:
I reformatted everything so that every sentence begins in a new line. This makes diffs easier to read I feel.